src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.card {
+  &__stars {
+    display: flex;
+    align-items: end;
+    justify-content: space-between;
+    line-height: 14px;
+
+    &-reviews {
+      margin: 0;
+      font-size: 10px;
+      color: $title-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
```

Comment:
Code Style issue: According to BEM methodology, `justify-content` property is not usually used with `space-between` value for `.stars` block. It's better to use `flex-start` or `center` to align stars.